### ISSUE TITLE
Fade out

### ISSUE THREAD

**Issue by khalidahmedshalabi** [Gammix/fader.inc#1]
Wouldn't it be better to have this code fade a textdraw out instead of returning an error? https://github.com/Gammix/fader.inc/blob/master/pawno/include/fader.inc#L29

``` PAWN
if (color == 0 && boxcolor == 0)
{
    // fade out code
}
```

**Comment by Gammix** [comment]
Are you saying to fade the textdraw from 256 opacity to 0 (from visible to invisible<hide>) ?

**Comment by khalidahmedshalabi** [comment]
Yeah, from whatever state to completely hidden (TextDrawHideForPlayer/All...etc)
